lib/node-integration-iframe.mustache.js

```diff
@@ -1,4 +1,8 @@
-(function () {
+(function (runtimeFlag) {
+  if (window[runtimeFlag] === true) {
```

Comment:
We prefer to avoid `runInParent` for this reason as well as it being difficult to clean up leaky tests. In our documentation, we suggest using `useIframe: false` instead